coremltools/converters/nnssa/coreml/graph_pass/mlmodel_passes.py

```diff
@@ -275,35 +298,104 @@ def _find_redundant_transposes(nn_spec):
         """
```

Comment:
```suggestion
"""
        Search the neural network spec for pairs of transposes that together
        are the identity, and return a list of those pairs. This will only find		        are the identity, and return a list of those pairs. This will only find
        redundant pairs that are adjacent in the topologically sorted layer		        redundant pairs that are adjacent in the topologically sorted layer
        list and will not find pairs in more complicated structure.		        list and will not find pairs in more complicated structure.
        """
```

comment should be updated now, as redundant transposes are now detected in a more complicated structure.